plugin/cache/cache.go

```diff
@@ -241,7 +244,7 @@ const (
 	defaultCap = 10000 // default capacity of the cache.
 
 	// Success is the class for caching positive caching.
-	Success = "success"
+	Success CacheClass = "success"
```

Comment:
Don't think `CacheClass` needs to be exported

Reply:
Done